pyxem/utils/subpixel_utils.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is code is very bad and I am deeply ashamed that I wrote and would thus appreciate it if we made the module private. Fortunately that would also be an API consistent decision

Reply:
Haha.  This code actually works fairly well now.  It never was going to work great as you can't really get much sub pixel accuracy in many cases.  If you really need better accuracy then increase your camera length/ decrease your convergence angle/aperture size...